aiida/backends/djsite/db/migrations/__init__.py

```diff
@@ -64,6 +68,7 @@ def _upgrade_schema_generation(version, apps, _):
 
 def upgrade_schema_version(up_revision, down_revision):
     from functools import partial
```

Comment:
weird, only because of the extra new line, does pylint start failing it for not having a docstring 🤷 (https://github.com/aiidateam/aiida-core/runs/3690148576)

```suggestion
def upgrade_schema_version(up_revision, down_revision):
    """Run migrations, to translate the database schema."""
    from functools import partial
```